README.md

```diff
@@ -1,6 +1,96 @@
 # example-django
+
 An example repository to demonstrate Django support in Pants.
 
-See [pantsbuild.org](https://www.pantsbuild.org/) for much more detailed documentation.
+The demo consists of a set of microservices that implement an elaborate "Hello, World" site.
+This repo shows how Pants can support multiple Django services in a single repo, with
+minimal copypasta.
+
+The layout in this repos enforces a strict distinction between Django apps (reusable units of
+functionality such as models and related views), vs. services ("projects" in Django parlance) -
+deployable services composed of multiple apps and typically attached to one or more database instances.
```

Comment:
There's a conceit here that wasn't immediately obvious to me, which I think arises from how Django people think about Django Apps: usually the app is a thing that is reused in the context of multiple projects, and each project has a database. Here it looks like the app and its corresponding database go hand-in-hand. This is _different_.

Reply:
I've clarified in the README, hopefully.